src/style.css

```diff
@@ -1 +1,69 @@
-/* Styles go here */
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.nav {
+  margin-right: 50px;
+}
+
+.nav__list {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  align-items: center;
+  text-transform: uppercase;
+  list-style: none;
+}
+
+.nav__item {
+  margin-left: 20px;
+  flex-shrink: 0;
+}
+
+.nav__item:nth-child(1) {
+  margin-left: 0;
+}
+
+.nav__link {
+  display: flex;
+  padding: 23px 0;
+  text-decoration: none;
+  color: #000;
+  cursor: pointer;
+}
+
+.is-active {
+  position: relative;
+  color: #00acdc;
+}
+
+.is-active::after {
+  content: "";
+  position: absolute;
+  bottom: 0;
+  height: 4px;
+  width: 100%;
+  border-radius: 8px;
+  background-color: #00acdc;
+}
+
+.image {
+  padding-left: 50px;
```

Comment:
this makes the space to the left of the logo clickable, which is not right

Reply:
![image](https://user-images.githubusercontent.com/43670243/167487227-2853d40e-dfe9-4f52-bf76-3b8e64b3f213.png)

I assigned margins to <a> but there was still this space below it
How can I get rid of it?